TravelPoints-UI/src/app/services/tourist-attraction.service.ts

```diff
@@ -0,0 +1,77 @@
MN83jWVvdDexa8b43ztQtWmWBSv9WohqGLlTM5DuoA2MSfEXFU6nUrW5pqpABb5bOHz3uNwdoxkia0wrMt0zF0pJdI+YuwO2LH7RY6eeUUFKqlD5j1OKVPZw7A9g1x4gnFszsO1qaVEFT/gJcXs1iPmwR7e8T6FBSoXNgT6nb+/P1H1c6Yu5Qre+9xc9QYl3cuLE5iCVO6qgCe43AwLPjy2XzmMt6tDT3H67UNNYBXSGJPSkkAEM5bfLZAg7T6iqWrpdOxJCg4J83xEHEkGakpCMgHAJbAJG38IZp9KtYKXtu6sYYlIGxHeDlJld7IpyK0EIJMxwzEtkVF2Zzh/EEylIoNurpB3Y3FwbDe/nMEaDQKYlxSUsU/LffBcWtEUqTQlw6cgPeogs742+ohOW1dgpgy5SimlKmVkOSrwbDNn27+XMVpehKkF2AqSRSpViXIsKrixDxLKKQFUgJIb+uzgDHtFdrKlBawVA1AYKak1YfbH0zDSt0NLYYuUSA4yc5b7P72xEnDpikqUlRKhdKiSFEjq7Xti/bEMUh5K+oujqDh6WBspzhu2c7tA+gmIPyqUCmoDHaojBB/kzbGKkriSw+ZJUTZDXZ7hR9d8nYXZogTOFTDc5csCCRh+4y12iNcypwpwzMDtuGDj7587smSlrKkqSSCSZa3SGdN2vfuxvv2hV5E+DvENYkFIW1VIdgDuRkh/PvCh0qfNSAAUeXJFwSNgXFo5FLZVX7HZH+hpLBIISxABBpLsQ63szMG9mjieD1PMmTCSWpAFg5wCXJ7YteJjOQk1KI7AOTfewsL2tBXMFvzte2c2Hq2BEuclwKgdYbpUCfIfLXYv6RAZBQ7EbGyT3OSAe+2GNwLRIuzEgMTs9mD57/S0SEDDuQMZ6iABh9tvG8CdCIebUCA9gCfT0OxAOReHSpZS1JUXLEW83Dj07O3iIQpXzACl8u5F+rpF1e/n2etSlON/lFwoO1nd+3baKrwA2ULmWv5gS4CRdL9JfYN6G/hzKmYmSgpSSSHJdqgG3OwGHbfMcloNKnpKwzEnP4nd3I3v/UyqQkJ672ZWDc2yBgYhSfkBs3S1CsGhZF3JNs4J7f1HaaXopYSXAKvxkWfvgufr9GEQr1yEqSWqTZLvg7BjZ23ttEsoqqKWUTuoiyXelI6QCPIzd4T1UAHMkMsGXdBBcpZkXChfHmwc/cmSlJbcg2yfQXF/+8RaiTNCSFpBD/Ml0M56bDAD2PjF4YqUSPmNOASHfux3t/Zhv3LkQ2cVBRsaQ71NjAa5dxfHrEiFlmDAl2CbA7gkEC/p39ImQkNe5FnLHLWBIx7fRoEVKSFdIpSRUstgXPzMyQCMbP6Mk0wEmfS6iovskvY7/wAMWz4MP1E5SSCR6i5U72YA97XiCYBWZbkqTghqiS4AU+5c/YwtNLURQpns4DF9ySCW7X8+I0pcganh09U9KQgqSshScO4SkKAuQ2FeBAXGtIUTACl08sdTVCxJAcm7HAp3F8OR8Ea1CdQULLOFBKiQwLsHWLCwIvub2gT4pVzZhmJUTdRFIKuWQ1JUrdJBG1qW7OJJKi3LYpNVpHWlJCSFrrQQUggEusZci5I3/KDlyEIQGLEOXIu5e4CWLhzc7QXopAVKUogKmISrqXlILu5s3geRnMV+hUp3KCEljV0e9kq7n6PEanJfb9kneVNUVU9SUoN2DKcKIFyXuGsNw8MkiaGrUkqSQMA2q6nLd3S7A+uYKTpVlAKgkLHyAMSSUXwQGuRm77ZgKXw+WtaldSSQHLWSXckN0uezkXJ3gTTW/wDwC6CuZJUoLsDYD8QILPbqGWY7xxE9ptKbBmuS2e5Pcece0A6YpQlSEdzfvSO9rdg92hh1MtJrqpY/MokPs4BN8dhjZ7S4Jqki09i3mVBVYJPQSkDfFy5z/LeK3iOpXWCopTKoBZVwonIakuxawb3gzU64FwGN0i4FnG598xLoUgyhUEquCAQFM1nfe3mCKSL+wDKM9BKlITTlVLqYN4chPk4s4ibUawTClBIAWMl+xFjbcDzeLErCUPZha21iwAGcMwv4s4C1epQUEhn2CrG5az4N7N47tArdOge2xDw+YoEoWlm6VXqAINOQBYi+MHxAkqqWtSSlgk2PWoMxILk2O++fSLrjHMCNOS5rBSkkgEBCkhRbsARa5LQHxXSpK0zQkm1BbBbqS7ENuHOHTiG6TryTRX6xT/q0pIe5c1BVTJwxZwBa21jDUL6BW9J6Tez3DPl7bsCRBRkAqQRcFi4cOz03CVM9W4bpFxaCAhISZVABwQ6cCwdu2fb2KeRUiJAGoCFl1yRMP7XTj94hva0KGL6TZLPeypiQbs7AAbZ/7BRovp/fskF/CnykE+S6cxM36s+AG8dSoUKB8fkZe6RI5abC4v8AVI/K0VqEjtghvF5mPoPpHYUc2PhlMrODIBSskAl03Pm5/IfSAJfzzf8AKOn/AC3SLdrWhQo7lzIg0EgdKjuKmO4dSXYxU6pRqUNuYsewAaFCjKPzMCymSkvLsLKtYWZJZvRg3pBk1Rqlh7dv3R/M/WFCjKXb8jJh/IwBxxRAlMWdTHz0LhQonH86/IDwOtI2CQw94rNWbagbBNhsOsiw9h9IUKNsfP8AeSWD6AdSjuFK/KOyC6JZNyQkk9+tvyhQo6Blnww9cs7ln+hieSoipiRcm3dzChRy5uGD4JpodE8G4CZdj5msftaIZElLJ6Rc3sNqmhQoX+QBpOoX+kKTUpqU2ct8yNvc/WLVKQEqazG3jra3tChQsnb8FIzvBT1K9VfdLn7xb8NWShJJJLDPon+f3hQo0zFRGBIIBIc1p/NJ/Mk+8GyA0u1mU1u1QhQohlR5CpKy6bm8tz5LEv62H0EUAN1/8n5AEQoUdEOGKXJpNUf9nl+EW8frjj6faBVKPJF/xo+5IMKFHI/mf92F3I5P/qI+6oB0UsJIpADzS7Buw/KFChR4kTI5q9QsKspQ3sSI7ChRqkqIP//Z"
```

Comment:
M: create a separate .ts file with constants where you are going to place these links